code/game/objects/effects/misc.dm

```diff
@@ -71,6 +71,14 @@
 	if(icon_state == "blocker")
 		icon_state = ""
 
+/obj/effect/forcefield/dense0forguns
+	resistance_flags = PROJECTILE_IMMUNE
+
+/obj/effect/forcefield/dense0forguns/Initialize(mapload)
+	. = ..()
+	if(icon_state == "blocker")
+		icon_state = ""
```

Comment:
```suggestion
```

Reply:
Doesn't removing this make the special forcefield visible ingame?

Reply:
no, it already gets inherited from parent